package.json

```diff
@@ -61,62 +61,62 @@
     "postcss-selector-parser": "2.2.3",
     "postcss-values-parser": "1.5.0",
     "regexp-util": "1.2.2",
-    "remark-math": "1.0.4",
+    "remark-math": "1.0.5",
     "remark-parse": "5.0.0",
-    "resolve": "1.5.0",
-    "semver": "5.4.1",
+    "resolve": "1.8.1",
+    "semver": "5.6.0",
     "string-width": "2.1.1",
-    "typescript": "3.2.0-rc",
+    "typescript": "3.3.0-dev.20181127",
     "typescript-estree": "6.0.0-rc.1",
     "unicode-regex": "2.0.0",
-    "unified": "6.1.6",
+    "unified": "6.2.0",
     "vnopts": "1.0.2",
-    "yaml": "1.0.0-rc.8",
-    "yaml-unist-parser": "1.0.0-rc.4"
+    "yaml": "1.0.2",
+    "yaml-unist-parser": "1.0.0"
   },
   "devDependencies": {
     "@babel/cli": "7.1.5",
-    "@babel/core": "7.1.5",
-    "@babel/preset-env": "7.1.5",
+    "@babel/core": "7.1.6",
+    "@babel/preset-env": "7.1.6",
     "babel-loader": "8.0.4",
     "benchmark": "2.1.4",
     "builtin-modules": "2.0.0",
-    "codecov": "2.2.0",
-    "cross-env": "5.0.5",
-    "eslint": "4.18.2",
-    "eslint-config-prettier": "2.9.0",
+    "codecov": "2.3.1",
+    "cross-env": "5.2.0",
+    "eslint": "4.19.1",
+    "eslint-config-prettier": "2.10.0",
     "eslint-friendly-formatter": "3.0.0",
-    "eslint-plugin-import": "2.9.0",
-    "eslint-plugin-prettier": "2.6.0",
-    "eslint-plugin-react": "7.7.0",
-    "execa": "0.10.0",
-    "jest": "23.3.0",
-    "jest-junit": "5.0.0",
+    "eslint-plugin-import": "2.14.0",
+    "eslint-plugin-prettier": "2.7.0",
+    "eslint-plugin-react": "7.11.1",
+    "execa": "0.11.0",
+    "jest": "23.6.0",
+    "jest-junit": "5.2.0",
     "jest-snapshot-serializer-ansi": "1.0.0",
     "jest-snapshot-serializer-raw": "1.1.0",
-    "jest-watch-typeahead": "0.1.0",
+    "jest-watch-typeahead": "0.2.0",
     "mkdirp": "0.5.1",
     "prettier": "1.15.2",
     "prettylint": "1.0.0",
     "rimraf": "2.6.2",
-    "rollup": "0.47.6",
+    "rollup": "0.67.3",
     "rollup-plugin-alias": "1.4.0",
-    "rollup-plugin-babel": "4.0.0-beta.4",
-    "rollup-plugin-commonjs": "8.2.6",
-    "rollup-plugin-json": "2.1.1",
-    "rollup-plugin-node-builtins": "2.0.0",
-    "rollup-plugin-node-globals": "1.1.0",
-    "rollup-plugin-node-resolve": "2.0.0",
+    "rollup-plugin-babel": "4.1.0-0",
+    "rollup-plugin-commonjs": "8.4.1",
+    "rollup-plugin-json": "2.3.1",
+    "rollup-plugin-node-builtins": "2.1.2",
+    "rollup-plugin-node-globals": "1.4.0",
+    "rollup-plugin-node-resolve": "2.1.1",
     "rollup-plugin-replace": "1.2.1",
     "rollup-plugin-uglify": "3.0.0",
-    "shelljs": "0.8.1",
-    "snapshot-diff": "0.4.0",
+    "shelljs": "0.8.3",
+    "snapshot-diff": "0.4.1",
     "strip-ansi": "4.0.0",
     "tempy": "0.2.1",
     "webpack": "3.12.0"
   },
   "resolutions": {
-    "@babel/code-frame": "7.0.0-beta.46"
```

Comment:
this will probably break your node 4 CI unless you transpile everything (maybe you do?)